python/paddle_fl/mpc/examples/model_encryption/README.md

```diff
@@ -0,0 +1,138 @@
+## Instructions for PaddleFL-MPC Model Encryption Demo
+
+([简体中文](./README_CN.md)|English)
+
+### 1. Introduction
+
+This document introduces how to run encrypt PaddlePaddle  model, then train or update encrypted model, or predict encrypted data with encrypted model. Model encryption is suitable for protecting training/prediction data and model.
+
+### 2. Scenarios
+
+Model encryption demo contains three scenarios:
+
+*  **Encrypt Model and Train**
+
+Each party loads PaddlePadlde model and encrypts it. Each party feeds the encrypted data to train the encrypted model. Each party can get one share for the encrypted model. PaddlePaddle model can be reconstructed with three encrypted model shares.
```

Comment:
clarify that in this case, the PaddlePaddle model is a new or empty model, and the transpile API is to make it a encrypted and empty model shares.

Reply:
fixed